Wrappers/Python/ccpi/optimisation/operators/DiagonalOperator.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#  CCP in Tomographic Imaging (CCPi) Core Imaging Library (CIL).
+
+#   Copyright 2017 UKRI-STFC
```

Comment:
I think it's 2020 ;)

Reply:
Yeah I wasn't sure whether to leave it as 2017 or change to current year. Somehow I don't like all the different files to have different years on from which year they happened to be written. I'd rather have an identical header in all files with the same year shown referring to the year of the software release perhaps.

Reply:
Agree changing all to 2020, or maybe 2017-2020. It will be ok with our pubs.